<launch>
	<arg name="marker_size" default="3.0" />
	<arg name="max_new_marker_error" default="0.04" />
	<arg name="max_track_error" default="0.1" />
	<arg name="cam_image_topic" default="/prosilica/image_rect_color" />
	<arg name="cam_info_topic" default="/prosilica/camera_info" />	
	<arg name="output_frame" default="/torso_lift_link" />

	<node name="ar_track_alvar" pkg="ar_track_alvar" type="individualMarkersNoKinect" respawn="false" output="screen" args="$(arg marker_size) $(arg max_new_marker_error) $(arg max_track_error) $(arg cam_image_topic) $(arg cam_info_topic) $(arg output_frame)" />
</launch>
